_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sercomm FG1000B.11
+has_children: false
+layout: default
+parent: Sercomm
+---
+
+# Hardware Specifications
+
+|                 |                        |
+| --------------- | ---------------------- |
+| Vendor/Brand    | Sercomm                |
+| Model           | FG1000B.11             |
+| Chipset         | BCM68360_B1            |
+| Flash           | NAND 128 MB            |
+| RAM             | 256 MB                 |
+| CPU             | Broadcom B53 Dual Core |
+| CPU Clock       | 1500MHz                |
+| Bootloader      | CFE                    |
+| Load addr       | 0x0000000000080000     |
+| 2.5GBaseT       | Yes                    |
+| PHY Ethernet    | RTL8221B               |
+| Optics          | SC/APC                 |
+| IP address      | 192.168.100.1/24       |
+| Web Gui         | ✅, no login needed    |
+| SSH             | No                     |
+| Telnet          | No                     |
+| Serial          | No                     |
+| Serial baud     | 115200                 |
+| Serial encoding | 8-N-1                  |
+| Form Factor     | ONT                    |
+
+
+{% include image.html file="fg1000b-11_rear.jpg" alt="Sercomm FG1000B.11" caption="Sercomm FG1000B.11 rear" %}
+{% include image.html file="fg1000b-11_bottom.jpg" alt="Sercomm FG1000B.11 bottom" caption="Sercomm FG1000B.11 bottom" %}
+{% include image.html file="fg1000b-11_side1.jpg" alt="Sercomm FG1000B.11 side 1" caption="Sercomm FG1000B.11 side 1" %}
+{% include image.html file="fg1000b-11_side2.jpg" alt="Sercomm FG1000B.11 side 2" caption="Sercomm FG1000B.11 side 2" %}
+
+# Serial
+
+See picture side2 for the pin identification, use 112500 8-N-1
+The ONT seems only to display output of the ROM CFE and flash CFE, but don't allow interupting the boot...
+
+dump:
+
+{% include serial_dump.html file="fg1000b-11_boot_cfe.txt" alt="Sercomm FG1000B.11 CFE boot dump" %}
+
+## Getting/Setting ONU GPON PLOAM password
+
+PLOAM can be set directly for Text or Hexa(without 0x) via Web interface if <10 digit otherwise POST call to URL allow > 10 digits for example 20 digit hex can be setup via:
+
+```
```

Comment:
couldn't a script be made in js? or are there cors problems?
otherwise, can't the unnecessary headers be removed?

Reply:
I'm not sure I understand the interest there ? You mean being able to set the PLOAM of your ONT from the GPON page using javascript XHR call ? Why would you want to do this?
Also for removing the unnecessary header I don't have the time to remove on by one and test with this firmware which is very picky...

Reply:
no a simple js script that makes an ajax request in the gpon hack wiki, such as the script to root huawei ma5671a or the one to generate the enable password for some ont cigs (nokia 010g-q,r,t, 010s-q, zte f6005, cig,...), or scripts to convert hex/ascii.

I can do it if you don't have time. for the removal of headers, it doesn't matter, a few more headers won't hurt.

Reply:
you can do a test with the following headers:

```
curl -i -s -k -X $'POST' \
-H $'Host: 192.168.100.1' -H $'Accept: application/json, text/javascript, */*' -H $'Content-Type: application/x-www-form-urlencoded' -H $'Origin: http://192.168.100.1' -H $'Referer: http://192.168.100.1/ONT/client/html/content/config/setup.html?lang=en' -H $'Connection: close'
    -b $'lang=en' \
    -d $'ploam_password=00000XXXXXXXXXXXXXXX' \
    $'http://192.168.100.1/ONT/client/data/Router.json'
```

and

```
curl -i -s -k -X $'POST' \
-H $'Host: 192.168.100.1' -H $'Content-Type: application/x-www-form-urlencoded' 
    -b $'lang=en' \
    -d $'ploam_password=00000XXXXXXXXXXXXXXX' \
    $'http://192.168.100.1/ONT/client/data/Router.json'
```

Reply:
integrated 2nd option.